### ISSUE TITLE
WET Menu System

### ISSUE THREAD

**Issue by mxlav** [wet-boew/wet-boew-drupal#550]
On a landing page, add a menu in the sidebar, select Wetkit Nav for styling

Expecting the menu items to not have bullets and the title to be in light grey box...not happening.

**Comment by sylus** [comment]
The wetkit nav style was just deprecated as of this week as no longer need it to get the menu theming I want applied. Not sure how your adding the menus? I assume you added the menus straight to the panel layout? What you might be missing is you instead have to use menu blocks and that has to be added to the panel layout. 

So:

a) Create a menu
b) Make the menu a menu block
c) Add the menu block to the panel region

This workflow is due to WET having very exacting markup and menu block allowing more theming functions on the menu subsystem.

**Comment by sylus** [comment]
The default mega menu you get in the left region is done this way and provides an example. Also menu block gives you a lot more options then the standard menu.

**Comment by mxlav** [comment]
@sylus, this only appears to work when the layout is WetKit - 2 column?

**Comment by sylus** [comment]
The menu style gets applied when under the wb-sec-def attribute this gets applied in 2 column layouts.

Are you trying to add menu's in other types of layouts? Could you be a bit more explicit on what you are trying to do :)

**Comment by mxlav** [comment]
I am currently trying to figure out what/how to design a few landing pages. So I am changing layouts from the available wetkit layouts and many of the available Panopoly layouts. What is happening is that the wetkit nav style is not consistant from one layout to another.

I was also wondering why some of the other WET-BOEW styling are not available as styling defaults such as `module-menu-section` ?

**Comment by sylus** [comment]
You should stick to just the WET layouts. Panopoly ones are included so can show how to extend the WetKit ones. If you select Panopoly layouts a lot of the CSS won't work because are not using WET classes.

As mentioned the wetkit nav style (brush style) is now deprecated. The menu should just work as is now but is meant to work with the 2 column layout as that has the wb-sec-def attribute no other layout does. This is part of WET and would have to be addressed with them as to why other layouts don't support the menu system.

Basically we support all layouts from WET itself. If you are able to design a layout using just WET markup then you will be able to do it from within Drupal.

There might be some confusion here on my part to what is trying to be accomplished.

@pjackson28 @laurawesley

**Comment by sylus** [comment]
Think this was further discussed here: https://github.com/wet-boew/wet-boew/issues/744

**Comment by sylus** [comment]
I have added a new 3 column panel layout that has full menu support. Should provide a good guide on how to make your own in reference to the wet-boew discussion above.

Hope this helps!
